modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// IPI
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+// * installing/ipi-install-post-installation-configuration.adoc
+// UPI
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * post_installation_configuration/post-install-bare-metal-configuration.adoc
+
+ifeval::["{context}" == "ipi-install-post-installation-configuration"]
+:postinstall-bare-metal-ipi:
+endif::[]
+ifeval::["{context}" == "post-install-bare-metal-configuration"]
+:postinstall-bare-metal-upi:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Optional: Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. Creating an additional `br-ex` bridge requires a change to the `configure-ovs.sh` shell script. Changing the shell script requires that you create a machine config manifest file for the network plugin of the Secure Neutron Networking implementation
```

Comment:
> Creating an additional `br-ex` bridge requires a change to the `configure-ovs.sh` shell script. Changing the shell script requires that you create a machine config manifest file for the network plugin of the Secure Neutron Networking implementation

Minor comment here might be that the change we are modifying doesn't directly update the configure-ovs.sh script so much as place a variable that if present is referenced by the script, and when present will allow for the creation of the secondary network. 

Suggested verbiage change (example):

> You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. Creating an additional network: `br-ex1` can be accomplished with a machine-config manifest file defining the definition of your target interface that will be used to host this bridge network interface.

Reply:
Thanks. How about:

> You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in the target interface definition of a machine config manifest file.